### ISSUE TITLE
IOS Flutter, build failing

### ISSUE THREAD

**Issue by superhumang** [razorpay/razorpay-flutter#57]
** BUILD FAILED **
Xcode's output:
↳
    ~/flutter/.pub-cache/hosted/pub.dartlang.org/razorpay_flutter-1.1.2/ios/Classes/RazorpayDelegate.swift:4:42: error: use of undeclared type 'RazorpayPaymentCompletionProtocolwithData'
    public class RazorpayDelegate: NSObject, RazorpayPaymentCompletionProtocolwithData, ExternalWalletSelectionProtocol {
                                             ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    ~/flutter/.pub-cache/hosted/pub.dartlang.org/razorpay_flutter-1.1.2/ios/Classes/RazorpayDelegate.swift:4:85: error: 'ExternalWalletSelectionProtocol' is unavailable: cannot find Swift declaration for this protocol
    public class RazorpayDelegate: NSObject, RazorpayPaymentCompletionProtocolwithData, ExternalWalletSelectionProtocol {
                                                                                        ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    Razorpay.ExternalWalletSelectionProtocol:2:17: note: 'ExternalWalletSelectionProtocol' has been explicitly marked unavailable here
    public protocol ExternalWalletSelectionProtocol {
                    ^
   ~/flutter/.pub-cache/hosted/pub.dartlang.org/razorpay_flutter-1.1.2/ios/Classes/RazorpayDelegate.swift:57:24: error: 'Razorpay' is unavailable: cannot find Swift declaration for this class
            let razorpay = Razorpay.initWithKey(key ?? "", andDelegateWithData: self)
                           ^~~~~~~~
    Razorpay.Razorpay:2:14: note: 'Razorpay' has been explicitly marked unavailable here
    public class Razorpay : NSObject {
                 ^

**Comment by superhumang** [comment]
This happed after updating xcode to the latest current version. It was working fine before

**Comment by essuraj** [comment]
yes it was working before and suddenly its not, not sure what the issue. (i probably cleaned the build)

**Comment by essuraj** [comment]
```
 /Volumes/DriveA/libs/flutter/.pub-cache/hosted/pub.dartlang.org/razorpay_flutter-1.1.2/ios/Classes/RazorpayDelegate.swift:4:42: error:
    'RazorpayPaymentCompletionProtocolWithData' is unavailable: cannot find Swift declaration for this protocol
    public class RazorpayDelegate: NSObject, RazorpayPaymentCompletionProtocolWithData, ExternalWalletSelectionProtocol {
                                             ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    Razorpay.RazorpayPaymentCompletionProtocolWithData:2:17: note: 'RazorpayPaymentCompletionProtocolWithData' has been explicitly marked unavailable here
    public protocol RazorpayPaymentCompletionProtocolWithData {
                    ^
    /Volumes/DriveA/libs/flutter/.pub-cache/hosted/pub.dartlang.org/razorpay_flutter-1.1.2/ios/Classes/RazorpayDelegate.swift:4:85: error:
    'ExternalWalletSelectionProtocol' is unavailable: cannot find Swift declaration for this protocol
    public class RazorpayDelegate: NSObject, RazorpayPaymentCompletionProtocolWithData, ExternalWalletSelectionProtocol {
                                                                                        ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    Razorpay.ExternalWalletSelectionProtocol:2:17: note: 'ExternalWalletSelectionProtocol' has been explicitly marked unavailable here
    public protocol ExternalWalletSelectionProtocol {
                    ^
    /Volumes/DriveA/libs/flutter/.pub-cache/hosted/pub.dartlang.org/razorpay_flutter-1.1.2/ios/Classes/RazorpayDelegate.swift:57:24: error: 'Razorpay' is unavailable:
    cannot find Swift declaration for this class
            let razorpay = Razorpay.initWithKey(key ?? "", andDelegateWithData: self)
                           ^~~~~~~~
    Razorpay.Razorpay:2:14: note: 'Razorpay' has been explicitly marked unavailable here
    public class Razorpay : NSObject {
                 ^
    note: Using new build system
    note: Building targets in parallel
    note: Planning build
    note: Constructing build description

Could not build the precompiled application for the device.
```

**Comment by superhumang** [comment]
Have mailed Razorpay Team regarding the same, let's see what went wrong! ive been trying my luck since yesterday to not avail, let me know if you happen to figure out a solution.

PS:- even Razorpay example app has the same error now!

**Comment by minhcasi** [comment]
same issue with latest XCode update 😅

**Comment by sharad-paghadal** [comment]
https://flutter.dev/docs/development/ios-project-migration

**Comment by superhumang** [comment]
Have already done that but to no avail, however build successfully without razorpay plugin

**Comment by Anmol92verma** [comment]
Same issues here, any solution ?

**Comment by superhumang** [comment]
RazorPay replied "We are checking the issue with our concerned team.
​
Please do give us some time to get back to you with a resolution of the issue.

Here to help :)
Tota Das" ..

In the meantime, i'm looking into integrating Paytm securely through platform channel or web!

**Comment by superhumang** [comment]
Reply from RazorPay:- 
We have not released our Flutter Plugin for Xcode11.4 still now. The integration kit is compatible with the Xcode11.3 version.
​Request you to use the Xcode version 11.3 for integration purposes. Sorry for the inconvenience caused.
​
Do let us know if you still have any issues regarding this integration, we will get back to you as soon as possible.

**Comment by nihalreddy33** [comment]
@superhumang did they update you anything regarding when they are planning to release the update for flutter plugin.

**Comment by superhumang** [comment]
Reply from RazorPay:-
As of now, there is no ETA of Flutter Plugin with Xcode 11.4 release.
Please use the 11.3 version, which is compatible with Razorpay Plugin.

**Comment by nihalreddy33** [comment]
Thank you @superhumang for the update.

**Comment by Nautiyalsachin** [comment]
Hi everyone, we have published the new release v1.1.3 with the fix of this issue, please feel free to update your plugin and let us know how it's working for you. Thanks!

**Comment by ananthpranav** [comment]
[!] Unable to determine Swift version for the following pods:

    - `razorpay_flutter` does not specify a Swift version and none of the targets (`Runner`) integrating it have the `SWIFT_VERSION` attribute set. Please contact the author or set the `SWIFT_VERSION` attribute in at least one of the targets that integrate this pod.

What do I do now?

**Comment by superhumang** [comment]
** BUILD FAILED **


Xcode's output:
↳
    Command CompileSwift failed with a nonzero exit code
    /Users/Developer/flutter/.pub-cache/hosted/pub.dartlang.org/razorpay_flutter-1.1.3/ios/Classes/RazorpayDelegate.swift:4:42: error:
    'RazorpayPaymentCompletionProtocolWithData' is unavailable: cannot find Swift declaration for this protocol
    public class RazorpayDelegate: NSObject, RazorpayPaymentCompletionProtocolWithData, ExternalWalletSelectionProtocol {
                                             ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    Razorpay.RazorpayPaymentCompletionProtocolWithData:2:17: note: 'RazorpayPaymentCompletionProtocolWithData' has been explicitly marked unavailable here
    public protocol RazorpayPaymentCompletionProtocolWithData {
                    ^
    /Users/Developer/flutter/.pub-cache/hosted/pub.dartlang.org/razorpay_flutter-1.1.3/ios/Classes/RazorpayDelegate.swift:4:85: error:
    'ExternalWalletSelectionProtocol' is unavailable: cannot find Swift declaration for this protocol
    public class RazorpayDelegate: NSObject, RazorpayPaymentCompletionProtocolWithData, ExternalWalletSelectionProtocol {
                                                                                        ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    Razorpay.ExternalWalletSelectionProtocol:2:17: note: 'ExternalWalletSelectionProtocol' has been explicitly marked unavailable here
    public protocol ExternalWalletSelectionProtocol {
                    ^
    /Users/Developer/flutter/.pub-cache/hosted/pub.dartlang.org/razorpay_flutter-1.1.3/ios/Classes/RazorpayDelegate.swift:57:24: error: use of
    unresolved identifier 'RazorpayCheckout'
            let razorpay = RazorpayCheckout.initWithKey(key ?? "", andDelegateWithData: self)
                           ^~~~~~~~~~~~~~~~
    note: Using new build system
    note: Building targets in parallel
    note: Planning build

**Comment by nihalreddy33** [comment]
@superhumang did you update the Xcode to 11.4?

**Comment by superhumang** [comment]
> @superhumang did you update the Xcode to 11.4?

Yes. The whole issue started after the update!

PS: Have securely incorporated paytm though! but still if i find a workable solution then will revert back to razorpay

**Comment by shashikantdurge** [comment]
Downgrading xcode to 11.3 fixed the problem for me. 

Does the new update fix issue with xcode 11.4 ?

**Comment by superhumang** [comment]
No, it does not, it is failing with the above mentioned output

**Comment by Nautiyalsachin** [comment]
@superhumang , @shashikantdurge, @nihalreddy33: Update your pod inside your ios folder, it should be 1.1.5, you can check it inside podfile.lock. If it doesn't match with it, please update it by running `pod update`. Also the package now should be v1.1.3. 

Let me know if it still not working. Thanks!

**Comment by superhumang** [comment]
Working, thanks

> @superhumang , @shashikantdurge, @nihalreddy33: Update your pod inside your ios folder, it should be 1.1.5, you can check it inside podfile.lock. If it doesn't match with it, please update it by running `pod update`. Also the package now should be v1.1.3.
> 
> Let me know if it still not working. Thanks!

**Comment by hprateek43** [comment]
Facing the same issue. Should we downgrade to 1.1.3 in podfile ?

**Comment by LOG-TAG** [comment]
the latest version should work on xcode 11.3.1??

**Comment by RohithKilaparthi** [comment]
Yes, 1.1.3 works with updated XCode. Do upgrade the pods if the build fails

**Comment by vikram94** [comment]
Xcode Version 11.2.1 (11B500). Building in Swift 5. Tried every version of razorpay_flutter. Not able to build 
Error:-
Module compiled with Swift 5.0 cannot be imported by the Swift 5.1.2 compiler: .../ios/Pods/razorpay-pod/Pod/Razorpay.framework/Modules/Razorpay.swiftmodule/x86_64.swiftmodule...

Please help. Tried deleting, updating Pods. nothing worked.

**Comment by vikram94** [comment]
Here is how I resolved the issue.. Hope helps someone and saves his time.

Put in Podfile   pod 'razorpay-pod', '1.1.5'
Delete Pod Folder
Put 'use_frameworks!'  Below target ‘Runner’ do
Type Command in iOS - Pod update razorpay-pod
flutter pub pub cache repair
Flutter build iOS

in case of an issue do ask

**Comment by ashishrawat2911** [comment]
> @superhumang , @shashikantdurge, @nihalreddy33: Update your pod inside your ios folder, it should be 1.1.5, you can check it inside podfile.lock. If it doesn't match with it, please update it by running `pod update`. Also the package now should be v1.1.3.
> 
> Let me know if it still not working. Thanks!

Thanks, it worked

**Comment by Siddharth-Dev** [comment]
@superhumang Did it worked for xcode v11.4 or above. I am having the same issue on Xcode11.4, I updated the pod version to 1.1.5 but still same issue.

**Comment by Harshalshree** [comment]
I still cannot resolve this error. Is anyone else facing the same?

**Comment by shameeksarkar** [comment]
Using Xcode 11.5 and facing the same issue. Any solution?

**Comment by vishwesh-aubergine** [comment]
I am using 1.1.3 and Xcode 11.5 facing the same issue @Nautiyalsachin

**Comment by Alok81234881** [comment]
Duplicate issue #92

**Comment by Bhardwajraju** [comment]
error: the following command failed with exit code 0 but produced no further output
    CompileC /Users/apple/Library/Developer/Xcode/DerivedData/Runner-fykulzvympdpxfggcxtxwljabmha/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/FMDB.build/Objects-normal/x86_64/FMDatabase.o /Users/apple/Downloads/androidproject/sanjar_creation/ios/Pods/FMDB/src/fmdb/FMDatabase.m normal x86_64 objective-c com.apple.compilers.llvm.clang.1_0.compiler
    /Users/apple/Downloads/flutter/.pub-cache/hosted/pub.dartlang.org/razorpay_flutter-1.2.1/ios/Classes/RazorpayDelegate.swift:57:24: error: use of unresolved identifier 'RazorpayCheckout'
            let razorpay = RazorpayCheckout.initWithKey(key ?? "", andDelegateWithData: self)
                           ^~~~~~~~~~~~~~~~
    note: Using new build system
    note: Planning build
    note: Constructing build description
    warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 4.3, but the range of supported deployment target versions is 8.0 to 13.2.99. (in target 'FMDB' from project 'Pods')

**Comment by karkoding** [comment]
I'm on Xcode 11.6 and updated razorpay pod to 1.1.5. Still no luck. Any found solution?

**Comment by abhi143u11** [comment]
I'm on Xcode 11.6 and updated the Razorpay pod to 1.2.2. Still no luck. Did anyone found a solution? My friend on VM machine its work with same thing and for me on MY MBP its not

**Comment by Nautiyalsachin** [comment]
@karkoding, @abhi143u11 : Can you guys go to [Your App Name] > ios > Podfile.lock and share the razorpay-pod version here? This will tell the current version of razorpay-pod which you are using in your project.

**Comment by Renatinaveen** [comment]
@Nautiyalsachin this is the pod version i'm on razorpay-pod (1.1.4)

**Comment by Nautiyalsachin** [comment]
Yes, it supports, razorpay-pod should be 1.1.5. I would suggest you to remove razorpay_flutter plugin and add it again. This will update razorpay-pod automatically to the latest version. @Renatinaveen Let me know if it works for you.

**Comment by Renatinaveen** [comment]
I have done the steps @Nautiyalsachin this is the output i'm getting

`Xcode's output:
↳
    /Users/smjsolutions/Documents/flutter/.pub-cache/hosted/pub.dartlang.org/razorpay_flutter-1.2.2/ios/Classes/RazorpayDelegate.swift:4:42: error: 'RazorpayPaymentCompletionProtocolWithData' is unavailable: cannot find Swift declaration for this protocol
    public class RazorpayDelegate: NSObject, RazorpayPaymentCompletionProtocolWithData, ExternalWalletSelectionProtocol {
                                             ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    Razorpay.RazorpayPaymentCompletionProtocolWithData:2:17: note: 'RazorpayPaymentCompletionProtocolWithData' has been explicitly marked unavailable here
    public protocol RazorpayPaymentCompletionProtocolWithData {
                    ^
    /Users/smjsolutions/Documents/flutter/.pub-cache/hosted/pub.dartlang.org/razorpay_flutter-1.2.2/ios/Classes/RazorpayDelegate.swift:4:85: error: 'ExternalWalletSelectionProtocol' is unavailable: cannot find Swift declaration for this protocol
    public class RazorpayDelegate: NSObject, RazorpayPaymentCompletionProtocolWithData, ExternalWalletSelectionProtocol {
                                                                                        ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    Razorpay.ExternalWalletSelectionProtocol:2:17: note: 'ExternalWalletSelectionProtocol' has been explicitly marked unavailable here
    public protocol ExternalWalletSelectionProtocol {
                    ^
    /Users/smjsolutions/Documents/flutter/.pub-cache/hosted/pub.dartlang.org/razorpay_flutter-1.2.2/ios/Classes/RazorpayDelegate.swift:57:24: error: use of unresolved identifier 'RazorpayCheckout'
            let razorpay = RazorpayCheckout.initWithKey(key ?? "", andDelegateWithData: self)
                           ^~~~~~~~~~~~~~~~`

**Comment by Renatinaveen** [comment]
> I have done the steps @Nautiyalsachin this is the output i'm getting
> 
> `Xcode's output: ↳ /Users/smjsolutions/Documents/flutter/.pub-cache/hosted/pub.dartlang.org/razorpay_flutter-1.2.2/ios/Classes/RazorpayDelegate.swift:4:42: error: 'RazorpayPaymentCompletionProtocolWithData' is unavailable: cannot find Swift declaration for this protocol public class RazorpayDelegate: NSObject, RazorpayPaymentCompletionProtocolWithData, ExternalWalletSelectionProtocol { ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ Razorpay.RazorpayPaymentCompletionProtocolWithData:2:17: note: 'RazorpayPaymentCompletionProtocolWithData' has been explicitly marked unavailable here public protocol RazorpayPaymentCompletionProtocolWithData { ^ /Users/smjsolutions/Documents/flutter/.pub-cache/hosted/pub.dartlang.org/razorpay_flutter-1.2.2/ios/Classes/RazorpayDelegate.swift:4:85: error: 'ExternalWalletSelectionProtocol' is unavailable: cannot find Swift declaration for this protocol public class RazorpayDelegate: NSObject, RazorpayPaymentCompletionProtocolWithData, ExternalWalletSelectionProtocol { ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ Razorpay.ExternalWalletSelectionProtocol:2:17: note: 'ExternalWalletSelectionProtocol' has been explicitly marked unavailable here public protocol ExternalWalletSelectionProtocol { ^ /Users/smjsolutions/Documents/flutter/.pub-cache/hosted/pub.dartlang.org/razorpay_flutter-1.2.2/ios/Classes/RazorpayDelegate.swift:57:24: error: use of unresolved identifier 'RazorpayCheckout' let razorpay = RazorpayCheckout.initWithKey(key ?? "", andDelegateWithData: self) ^~~~~~~~~~~~~~~~`

after adding Enable bitcode this error is gone. But i couldn't run project still getting this error You must rebuild it with bitcode enabled (Xcode setting ENABLE_BITCODE), obtain an updated library from the vendor, or disable bitcode for this target. @Nautiyalsachin.

**Comment by Nautiyalsachin** [comment]
Please share the **razorpay-pod** version here from **pofile.lock**

**Comment by Renatinaveen** [comment]
- razorpay-pod (1.1.4)
  - razorpay_flutter (1.1.5):
    - Flutter
    - razorpay-pod

**Comment by abhi143u11** [comment]
To solve the issue, in Podfile, under target 'Runner' add

pod 'razorpay-pod', :git => 'https://github.com/razorpay/razorpay-pod.git', :commit => '4159c5044489dd26687f688ddea44e80fb530e52'
update SWIFT_VERSION to 5.2
flutter clean and remove razorpay-pod in ios/Pods
pod install and run

I run the same and mine worked fine.


I added this line in podfile in ios folder as follows 

  # Keep pod path relative so it can be checked into Podfile.lock.
  pod 'Flutter', :path => 'Flutter'
#thislineadded
  pod 'razorpay-pod', :git => 'https://github.com/razorpay/razorpay-pod.git', :commit => '4159c5044489dd26687f688ddea44e80fb530e52'
  # Plugin Pods

**Comment by Renatinaveen** [comment]
> To solve the issue, in Podfile, under target 'Runner' add
> 
> pod 'razorpay-pod', :git => 'https://github.com/razorpay/razorpay-pod.git', :commit => '4159c5044489dd26687f688ddea44e80fb530e52'
> update SWIFT_VERSION to 5.2
> flutter clean and remove razorpay-pod in ios/Pods
> pod install and run
> 
> I run the same and mine worked fine. I am using Xcode 11.6 and previously getting the above issue
> 
> I added this line in podfile in ios folder as follows
> 
> # Keep pod path relative so it can be checked into Podfile.lock.
> pod 'Flutter', :path => 'Flutter'
> #thislineadded
> pod 'razorpay-pod', :git => 'https://github.com/razorpay/razorpay-pod.git', :commit => '4159c5044489dd26687f688ddea44e80fb530e52'
> 
> # Plugin Pods

@abhi143u11 after doing the following steps and when running pod install i'm getting this 
Pre-downloading: `razorpay-pod` from `https://github.com/razorpay/razorpay-pod.git`, commit `4159c5044489dd26687f688ddea44e80fb530e52`
[!] CocoaPods could not find compatible versions for pod "razorpay-pod":
  In snapshot (Podfile.lock):
    razorpay-pod (= 1.1.4)

  In Podfile:
    razorpay-pod (from `https://github.com/razorpay/razorpay-pod.git`, commit `4159c5044489dd26687f688ddea44e80fb530e52`)

None of your spec sources contain a spec satisfying the dependencies: `razorpay-pod (from `https://github.com/razorpay/razorpay-pod.git`, commit `4159c5044489dd26687f688ddea44e80fb530e52`), razorpay-pod (= 1.1.4)`.

**Comment by Renatinaveen** [comment]
> To solve the issue, in Podfile, under target 'Runner' add
> 
> pod 'razorpay-pod', :git => 'https://github.com/razorpay/razorpay-pod.git', :commit => '4159c5044489dd26687f688ddea44e80fb530e52'
> update SWIFT_VERSION to 5.2
> flutter clean and remove razorpay-pod in ios/Pods
> pod install and run
> 
> I run the same and mine worked fine. I am using Xcode 11.6 and previously getting the above issue
> 
> I added this line in podfile in ios folder as follows
> 
> # Keep pod path relative so it can be checked into Podfile.lock.
> pod 'Flutter', :path => 'Flutter'
> #thislineadded
> pod 'razorpay-pod', :git => 'https://github.com/razorpay/razorpay-pod.git', :commit => '4159c5044489dd26687f688ddea44e80fb530e52'
> 
> # Plugin Pods

Okay done now the version is razorpay-pod (1.1.5)
but i'm facing this error when bitcode is enabled. You must rebuild it with bitcode enabled (Xcode setting
    ENABLE_BITCODE), obtain an updated library from the vendor, or disable bitcode for this target.

**Comment by Bhardwajraju** [comment]
Tis is my podfile.lock    and mu Xcode version 11.3.1
PODS:
  - Flutter (1.0.0)
  - flutter_plugin_android_lifecycle (0.0.1):
    - Flutter
  - flutter_read_more_text (0.0.1):
    - Flutter
  - fluttertoast (0.0.2):
    - Flutter
  - FMDB (2.7.5):
    - FMDB/standard (= 2.7.5)
  - FMDB/standard (2.7.5)
  - image_cropper (0.0.2):
    - Flutter
    - TOCropViewController (~> 2.5.2)
  - image_picker (0.0.1):
    - Flutter
  - path_provider (0.0.1):
    - Flutter
  - path_provider_linux (0.0.1):
    - Flutter
  - path_provider_macos (0.0.1):
    - Flutter
  - razorpay-pod (1.1.4)
  - razorpay_flutter (1.1.5):
    - Flutter
    - razorpay-pod
  - share (0.0.1):
    - Flutter
  - shared_preferences (0.0.1):
    - Flutter
  - shared_preferences_linux (0.0.1):
    - Flutter
  - shared_preferences_macos (0.0.1):
    - Flutter
  - shared_preferences_web (0.0.1):
    - Flutter
  - sqflite (0.0.1):
    - Flutter
    - FMDB (~> 2.7.2)
  - TOCropViewController (2.5.2)
  - url_launcher (0.0.1):
    - Flutter
  - url_launcher_linux (0.0.1):
    - Flutter
  - url_launcher_macos (0.0.1):
    - Flutter
  - url_launcher_web (0.0.1):
    - Flutter
  - webview_flutter (0.0.1):
    - Flutter

DEPENDENCIES:
  - Flutter (from `Flutter`)
  - flutter_plugin_android_lifecycle (from `.symlinks/plugins/flutter_plugin_android_lifecycle/ios`)
  - flutter_read_more_text (from `.symlinks/plugins/flutter_read_more_text/ios`)
  - fluttertoast (from `.symlinks/plugins/fluttertoast/ios`)
  - image_cropper (from `.symlinks/plugins/image_cropper/ios`)
  - image_picker (from `.symlinks/plugins/image_picker/ios`)
  - path_provider (from `.symlinks/plugins/path_provider/ios`)
  - path_provider_linux (from `.symlinks/plugins/path_provider_linux/ios`)
  - path_provider_macos (from `.symlinks/plugins/path_provider_macos/ios`)
  - razorpay_flutter (from `.symlinks/plugins/razorpay_flutter/ios`)
  - share (from `.symlinks/plugins/share/ios`)
  - shared_preferences (from `.symlinks/plugins/shared_preferences/ios`)
  - shared_preferences_linux (from `.symlinks/plugins/shared_preferences_linux/ios`)
  - shared_preferences_macos (from `.symlinks/plugins/shared_preferences_macos/ios`)
  - shared_preferences_web (from `.symlinks/plugins/shared_preferences_web/ios`)
  - sqflite (from `.symlinks/plugins/sqflite/ios`)
  - url_launcher (from `.symlinks/plugins/url_launcher/ios`)
  - url_launcher_linux (from `.symlinks/plugins/url_launcher_linux/ios`)
  - url_launcher_macos (from `.symlinks/plugins/url_launcher_macos/ios`)
  - url_launcher_web (from `.symlinks/plugins/url_launcher_web/ios`)
  - webview_flutter (from `.symlinks/plugins/webview_flutter/ios`)

SPEC REPOS:
  trunk:
    - FMDB
    - razorpay-pod
    - TOCropViewController

EXTERNAL SOURCES:
  Flutter:
    :path: Flutter
  flutter_plugin_android_lifecycle:
    :path: ".symlinks/plugins/flutter_plugin_android_lifecycle/ios"
  flutter_read_more_text:
    :path: ".symlinks/plugins/flutter_read_more_text/ios"
  fluttertoast:
    :path: ".symlinks/plugins/fluttertoast/ios"
  image_cropper:
    :path: ".symlinks/plugins/image_cropper/ios"
  image_picker:
    :path: ".symlinks/plugins/image_picker/ios"
  path_provider:
    :path: ".symlinks/plugins/path_provider/ios"
  path_provider_linux:
    :path: ".symlinks/plugins/path_provider_linux/ios"
  path_provider_macos:
    :path: ".symlinks/plugins/path_provider_macos/ios"
  razorpay_flutter:
    :path: ".symlinks/plugins/razorpay_flutter/ios"
  share:
    :path: ".symlinks/plugins/share/ios"
  shared_preferences:
    :path: ".symlinks/plugins/shared_preferences/ios"
  shared_preferences_linux:
    :path: ".symlinks/plugins/shared_preferences_linux/ios"
  shared_preferences_macos:
    :path: ".symlinks/plugins/shared_preferences_macos/ios"
  shared_preferences_web:
    :path: ".symlinks/plugins/shared_preferences_web/ios"
  sqflite:
    :path: ".symlinks/plugins/sqflite/ios"
  url_launcher:
    :path: ".symlinks/plugins/url_launcher/ios"
  url_launcher_linux:
    :path: ".symlinks/plugins/url_launcher_linux/ios"
  url_launcher_macos:
    :path: ".symlinks/plugins/url_launcher_macos/ios"
  url_launcher_web:
    :path: ".symlinks/plugins/url_launcher_web/ios"
  webview_flutter:
    :path: ".symlinks/plugins/webview_flutter/ios"

SPEC CHECKSUMS:
  Flutter: 0e3d915762c693b495b44d77113d4970485de6ec
  flutter_plugin_android_lifecycle: dc0b544e129eebb77a6bfb1239d4d1c673a60a35
  flutter_read_more_text: b88a1a4b1fb97a8b79f5b0c57f6adbc618385fc7
  fluttertoast: 37ffabf5aeada07e7449f1df4b1cede0a09b5005
  FMDB: 2ce00b547f966261cd18927a3ddb07cb6f3db82a
  image_cropper: ac1e7cf980bd70366009264eef9d0dde9e87c0ba
  image_picker: 9c3312491f862b28d21ecd8fdf0ee14e601b3f09
  path_provider: 310712a70965ff5a8f3973c7696680e3c714855c
  path_provider_linux: 8ccefb8ca27b06685e4cf008a40077656de037a7
  path_provider_macos: 81cf01de1cd42c80e034d53d569c6893d141e529
  razorpay-pod: 592624080178be4c6121216508e7857fc245f4b8
  razorpay_flutter: bf22a7c0e12d9f906b6a493f546ee9afe7f8b508
  share: 49e00805ff093c6547819af71117dfbc9ddb2451
  shared_preferences: 9e0c9e3d3c131c89ba473e168357ead96cc6917a
  shared_preferences_linux: 609a60078bbeba2e51dfa847323bb42e10605cfe
  shared_preferences_macos: 5cbacae7bfe877667ad184c412de9e535b5befff
  shared_preferences_web: 109fd5d827274ef686f0f11c4038f5e3b6d6e893
  sqflite: bf214c78403cc57c2365b043d172b0ef290ad9fc
  TOCropViewController: e9da34f484aedd4e5d5a8ab230ba217cfe16c729
  url_launcher: fc44accb11b323eef0e5bad3b9f72ca242dd6837
  url_launcher_linux: 330c97ed67eb8c0194ef1a39e79b0b20563a23cb
  url_launcher_macos: 79606443bda05e56a09edd1af025db6fb8b2be9c
  url_launcher_web: 1317c1804d0af8b1060788448fc2c0a22cf3b307
  webview_flutter: a949291df549148954b1e2588cd7d97390fa6006

PODFILE CHECKSUM: 6e4bcf017fe79cdb73aa0fb05e02150bdf08132d

COCOAPODS: 1.9.3

**Comment by Nautiyalsachin** [comment]
@Bhardwajraju : As I can see in your code, you are using **razorpay-pod (1.1.4)**. Please remove the **razorpay plugin** and add it again, this will also automatically update the **razorpay-pod**

**Comment by Nautiyalsachin** [comment]
Hey @Renatinaveen, our SDK is already **bitcode enabled**. Please check for other plugins which you are using in your code.

**Comment by Renatinaveen** [comment]
> To solve the issue, in Podfile, under target 'Runner' add
> 
> pod 'razorpay-pod', :git => 'https://github.com/razorpay/razorpay-pod.git', :commit => '4159c5044489dd26687f688ddea44e80fb530e52'
> update SWIFT_VERSION to 5.2
> flutter clean and remove razorpay-pod in ios/Pods
> pod install and run
> 
> I run the same and mine worked fine. I am using Xcode 11.6 and previously getting the above issue
> 
> 
> I added this line in podfile in ios folder as follows 
> 
>   # Keep pod path relative so it can be checked into Podfile.lock.
>   pod 'Flutter', :path => 'Flutter'
> #thislineadded
>   pod 'razorpay-pod', :git => 'https://github.com/razorpay/razorpay-pod.git', :commit => '4159c5044489dd26687f688ddea44e80fb530e52'
>   # Plugin Pods
> 

@Bhardwajraju this solved my problem. Try this

**Comment by Renatinaveen** [comment]
> Hey @Renatinaveen, our SDK is already **bitcode enabled**. Please check for other plugins which you are using in your code. 

Thanks it's solved with the latest pod version. It's working fine now.

**Comment by leorachit** [comment]
Having this issue, please help.
